cores/esp32/esp32-hal-misc.c

```diff
@@ -209,9 +209,8 @@ bool verifyRollbackLater() { return false; }
 #endif
 
 #ifdef CONFIG_BT_ENABLED
-//overwritten in esp32-hal-bt.c
-bool btInUse() __attribute__((weak));
```

Comment:
@SuGlider I am wondering if this change is also applicable for other weak symbols in this file?

cc @sanketwadekar

Reply:
I have verified it. 
The issue occurs when a weak and strong symbols are part of the same `core.a` library file. 
In the building process with Arduino, it first compiles all the Core Files and build a reusable Library.
The other weak functions are changed/redefined in the Sketch, therefore, its strong symbol are correctly linked.

The issue only happens with this very specific `btInUse()` function because its strong symbol is in the same `core.a` file and it is ignored by the RISC-V Linker.